app/assets/stylesheets/administrate/components/_form.scss

```diff
@@ -11,15 +15,13 @@
   label {
     @include data-label;
     @include span-columns(2 of 12);
-    padding: $form-input-padding 0;
+    padding: ($base-spacing / 3) 0;
```

Comment:
`$base-spacing / 3` should be written without spaces around the operator: `$base-spacing/3`